scripts/packaging/createupmpackages.ps1

```diff
@@ -159,11 +159,29 @@ foreach ($entry in $packages.GetEnumerator()) {
         $registryName = "the $OutputTarget registry"
     }
 
-    # The examples folder needs a custom setup step
-    if ($entry.Name -eq "examples") {
-        Write-Output "Preparing the examples folder"
+    $samplesFolder = "$packagePath\Samples~"

```

Comment:
Is there significance to the ~ (i.e. wildcard?)

Reply:
Unity ignores folders that end in `~`. They recommend that as the name of the samples folder so it doesn't try to get loaded by the editor when the package is loaded, allowing it to be imported as a sample to the Assets folder without duplicate scripts and clashing GUIDs.